aws/requests.yaml

```diff
@@ -1,5 +1,5 @@
 releases:
-- name: "> 16.1.1 > 16.3.1 > 16.4.1 > 17.1.0 > 17.2.0"
+- name: "> 17.2.0"
```

Comment:
Will this make sure kube-state metrics 1.9.0 or higher will end up in lets say release 16.4.2 ?

Reply:
No it doesn't. As mentioned in the PR description I only fixed the patterns to what they are actually doing, not to what they were intended to do. So there already may be some 16.x releases which are not applying to this request. Implementing what this pattern was intended to do could break already released versions.

Reply:
No, you'll need to use `||` to define multiple ranges

Reply:
I added another commit which ensures what you requested :)